tests/kafkatest/services/zookeeper.py

```diff
@@ -33,12 +34,19 @@ class ZookeeperService(Service):
             "collect_default": False}
     }
 
-    def __init__(self, context, num_nodes):
+    def __init__(self, context, num_nodes, zk_sasl = False):
         """
         :type context
         """
+        self.kafka_opts = ""
+        self.zk_sasl = zk_sasl
         super(ZookeeperService, self).__init__(context, num_nodes)
 
+    @property
+    def security_config(self):
+        return SecurityConfig(zk_sasl=self.zk_sasl, sasl_mechanism='PLAIN')
```

Comment:
I'm a bit confused about this. Who is using SASL/PLAIN here? Kafka doesn't support it and I couldn't find any information regarding SASL/PLAIN for ZK (and the JAAS file introduced in this test seems to be using DIGEST-MD5 although it has `plain` in its name). What am I missing?